src/include/function/built_in_vector_functions.h

```diff
@@ -27,6 +27,14 @@ class BuiltInVectorFunctions {
     static uint32_t getCastCost(
         common::LogicalTypeID inputTypeID, common::LogicalTypeID targetTypeID);
 
+    inline void addFunction(std::string name, function::vector_function_definitions definitions) {
```

Comment:
move to cpp. I start realizing it's better to move more code into cpp file if the header being included in many places. So we don't need to recompile a lot of files when the design changes